seed/contentfulSync.ts

```diff
@@ -1,49 +1,45 @@
-import { PrismaClient } from '@prisma/client'
-
-const prisma = new PrismaClient()
-
-type ContentSyncResult = {
-  entriesCount: number
-  createEntriesCount: number
-  updateEntriesCount: number
-  deletedEntriesCount: number
-}
+import { PrismaService } from '../src/prisma/prisma.service'
+import { ContentfulService } from '../src/vendors/contentful/contentful.service'
+import { ContentService } from '../src/content/services/content.service'
+import { ProcessTimersService } from '../src/shared/services/process-timers.service'
 
 async function seedContentful() {
-  console.log('🔄 Syncing Contentful content...')
+  console.log('🌱 Syncing Contentful content...')
 
-  const API_URL = 'http://localhost:3000/v1/content/sync'
+  const prismaService = new PrismaService()
+  await prismaService.onModuleInit()
 
   try {
-    const response = await fetch(API_URL)
-
-    if (!response.ok) {
-      throw new Error(`HTTP ${response.status}: ${response.statusText}`)
-    }
-
-    const result = (await response.json()) as ContentSyncResult
-
-    console.log('✅ Contentful sync complete!')
-    console.log(`   Total entries: ${result.entriesCount}`)
-    console.log(`   Created: ${result.createEntriesCount}`)
-    console.log(`   Updated: ${result.updateEntriesCount}`)
-    console.log(`   Deleted: ${result.deletedEntriesCount}`)
-  } catch (error) {
-    if (error instanceof TypeError && error.message.includes('fetch')) {
-      throw new Error(
-        `Cannot connect to ${API_URL}\nMake sure the API server is running (npm run start:dev)`,
-      )
-    }
-    throw error
+    const contentfulService = new ContentfulService()
+    const processTimersService = new ProcessTimersService()
+    const contentService = new ContentService(
+      contentfulService,
+      processTimersService,
+    )
+
+    // We need to manually inject Prisma service
+    // @ts-expect-error - Accessing private property for manual DI
+    contentService._prisma = prismaService
+    contentService.onModuleInit()
+
+    const { entries, createEntries, updateEntries, deletedEntries } =
+      await contentService.syncContent()
```

Comment:
This is cool.

But, there's a dev workflow when working on DB models, where we want to run `npm run migrate:reset` over and over again. This'll _severely_ slow down that flow if it runs w/ ever migrate.

Can we do something similar to the `npm run seed:*` tasks that we have where we can pass prisma a command line arg to disable this seed if we so choose? 🤔 

https://github.com/thegoodparty/gp-api/blob/d63951d1b0d65470de6f732278ddbd9b0a53fc85/package.json#L22-L23